# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/tools/layout-debug/src/nsLayoutDebuggingTools.cpp
# Commit: 86e9449aba35
# Full Hash: 86e9449aba35c7439e8a41a2804388994462c00d
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r c7d1d0cfaa6c -r 86e9449aba35 layout/tools/layout-debug/src/nsLayoutDebuggingTools.cpp
--- a/layout/tools/layout-debug/src/nsLayoutDebuggingTools.cpp	Wed Jun 22 10:30:03 2022 +0000
+++ b/layout/tools/layout-debug/src/nsLayoutDebuggingTools.cpp	Wed Jun 22 10:42:34 2022 +0000
@@ -255,7 +255,7 @@
 nsLayoutDebuggingTools::DumpCounterManager() {
   NS_ENSURE_TRUE(mDocShell, NS_ERROR_NOT_INITIALIZED);
   if (PresShell* presShell = GetPresShell(mDocShell)) {
-    presShell->FrameConstructor()->CounterManager()->Dump();
+    presShell->FrameConstructor()->GetContainStyleScopeManager().DumpCounters();
   }
   return NS_OK;
 }